fhir-client/src/main/java/com/ibm/fhir/client/impl/FHIRClientImpl.java

```diff
@@ -767,15 +768,16 @@ protected synchronized Client getClient() throws Exception {
             // Add a hostname verifier if we're using an ssl transport.
             if (usingSSLTransport() && !isHostnameVerificationEnabled()) {
                 cb = cb.hostnameVerifier(new HostnameVerifier() {
+                    @Override
```

Comment:
it's not really an override? it's a specific implementation

Reply:
en... interesting... this was not added by me, let me try to remove it and see

Reply:
can not be removed, it's always be added back automatically. and the function is abstract, so seems have to override.